oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/Analyzer.java

```diff
@@ -247,8 +248,8 @@ private void init() {
     private void createMetric(final ScopeType scopeType,
                               final String dataType,
                               final DownsamplingType downsamplingType) {
-        String functionName = String.format(
-            "%s%s", downsamplingType.toString().toLowerCase(), StringUtils.capitalize(dataType));
+        String downSamplingStr = CaseUtils.toCamelCase(downsamplingType.toString().toLowerCase(), false, '_');
+        String functionName = String.format("%s%s", downSamplingStr, StringUtils.capitalize(dataType));
```

Comment:
I think this changed the format. Could you explain why?

Reply:
If we combine the downsampling(`SUM_PER_MIN`) and dataType(`labeled`), the function name would be `sum_per_minLabeled`. The format of the name is not consistent, so I change the downsampling to camel format. 
After the change, the name is `sumPerMinLabeled`.

Reply:
I am feeling this change impacts the existing function names? Most of e2e are failing.

Reply:
It only affects the `SUM_PER_MIN` downsampling. The e2e are failing because I cancel it. The `Build Tar` CI is running timeout, so I want to re-run it.

Reply:
I have clicked the `cancel workflow` button, but It's doesn't work. Maybe I need to wait a while.